benchmarks/templates/benchmarks/model.html

```diff
@@ -2,17 +2,26 @@
 
 {% block content %}
 <section class="individual_model container center">
-    <h1 class="title">{{ model.name }}</h1>
-    {% if model.reference_identifier %}
-        <a href="{{ model.reference_link }}">{{ model.reference_identifier }}</a>
+    {% if model.public %}
+        <h1 class="title">{{ model.name }}</h1>
+        {% if model.reference_identifier %}
+            <a href="{{ model.reference_link }}">{{ model.reference_identifier }}</a>
+        {% endif %}
+    {% else %}
+        <h1 class="title">Anonymous Submission</h1>
+        <span class='information'>This model was submitted to the 2022 competition, but was not made public.</span><br>
     {% endif %}
 
     {% if submission_details_visible %}
         <div class="box">
-            <span class='fine_print'>The following information is only visible to you:</span><br><br>
+            <span class='fine_print'>The following information is only visible to you (the model owner) and the site admins, Martin and Mike:</span><br><br>
```

Comment:
change to "The following information is only visible to you as the model owner:" (cut the part about site admins and especially the names)